scripts/helpers.js

```diff
@@ -1,16 +1,39 @@
+var moment = require('moment');
 var urllib = require('url');
 
 var utils = require('../lib/utils');
-
 var pkg = require('../package');
+
+var isUrl = utils.isUrl;
+var urljoin = utils.urljoin;
+
 var aframeCurrentSha = 'master';
 try {
   aframeCurrentSha = pkg.dependencies.aframe.split('#')[1];
-} catch (e) {
-}
+} catch (e) {}
 
-var isUrl = utils.isUrl;
-var urljoin = utils.urljoin;
+hexo.extend.helper.register('blog_attribution', function (author) {
+  // Twitter handle.
+  if (author[0] === '@') {
+    return '<a class="blog-attribution" href="https://twitter.com/' + author + '">' +
```

Comment:
don't you want this to be `author.substr(1)`? (I actually didn't know https://twitter.com/@cvanw works, heh)